backend/src/email/utils/callback/parsers/ses.ts

```diff
@@ -183,10 +181,37 @@ const parseEvent = async (
 }
 
 const parseRecord = async (record: SesRecord): Promise<void> => {
-  if (!(record.SignatureVersion === '1' && (await validateSignature(record)))) {
-    throw new Error(`Invalid record`)
-  }
   const message = JSON.parse(record.Message)
+  const {
+    unique_args: { message_id: id, message_hash: hash },
+  } = getSmtpApiHeader(message) as SmtpApiHeader
+  if (!id) {
+    logger.info({
+      message: 'No reference message id found',
+      id,
+      timestamp: record.Timestamp,
+    })
+    throw new Error('No reference message id')
+  }
+
+  const password = id + config.get('emailCallback.hashSecret')
```

Comment:
Rather than using the secret this way, why not generate a proper HMAC.

```ts
const secret = config.get('emailCallback.hashSecret')
const hash = crypto.createHmac('sha256', secret).update(`${id}`).digest('hex')
```